src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -4,12 +4,13 @@
 
 using System;
 using System.Reflection;
+using System.Runtime.ExceptionServices;
 
 namespace System
 {
-    public partial class AppDomain
+    public partial class AppDomain : System.MarshalByRefObject
```

Comment:
Nit: There's no need to specify `System` here, it should be `AppDomain : MarshalByRefObject`